code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -79,15 +79,16 @@
 	else
 		if ((world.time - timeofdeath) > decompose_time)
 			dust()
-			
+
 /mob/living/simple_animal/mouse/Destroy()
 	SSmob.all_mice -= src
-		
+
 	return ..()
-	
+
 //Pixel offsetting as they scamper around
 /mob/living/simple_animal/mouse/Move()
-	if(..())
+	.=..()

```

Comment:
not setting the return value on this properly was an error of mine in the past. i noticed it when it came to ladders